phippery/negbinom.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Uri Laserson
```

Comment:
We can probably put your name here, now :)

Reply:
I got rid of that whole copyright/license thing and followed what you did in modeling.py:
"""
@File: negbinom.py

@Author: Kevin Sung
"""